systers_portal/systers_portal/urls.py

```diff
@@ -29,6 +29,7 @@
         name='ckeditor_browse'),
     url(r'^contact/$', ContactView.as_view(), name='contact'),
     url(r'^about-us/$', AboutUsView.as_view(), name='about-us'),
+    url(r'^meetup/', include('meetup.urls')),
```

Comment:
Good approach! I would just suggest moving it to the top patterns, maybe right before including users patterns. All the app url patterns would look nicer as a batch.